### PR TITLE
Add LRO to proto annotations in test resources

### PR DESCRIPTION
**Pull request by andreamlin**

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto [hunk 1]
```diff
@@ -773,78 +781,78 @@ enum TopLevelEnum {
...
-    repeated bytes optional_repeated_bytes = 78;
-    repeated InnerMessage optional_repeated_message = 79;
-    repeated string optional_repeated_resource_name = 80;
-    repeated string optional_repeated_resource_name_oneof = 81;
-    repeated string optional_repeated_resource_name_common = 84;
-    repeated fixed32 optional_repeated_fixed32 = 82;
-    repeated fixed64 optional_repeated_fixed64 = 83;
-
-    map<int32, string> optional_map = 91;
+  // For testing all types, plus resource-names, as required and optional.
```
Comment (andreamlin) [inline src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto]: The 4-space indentation had been bugging me.

File: src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto [hunk 2]
```diff
@@ -763,8 +771,8 @@ message FindRelatedBooksResponse {
 
 // Metadata for a longrunning operation.
 message GetBigBookMetadata {
-    // Approximate percentage of the book processed thus far.
-    int32 progress_percent = 1;
+  // Approximate percentage of the book processed thus far.
```
Comment (andreamlin) [inline src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto]: The rest of the diff is just to remove the 4-space indentation that had been bugging me.

File: src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto [hunk 3]
```diff
@@ -264,6 +268,9 @@ service LibraryService {
     option (google.api.method_signature) = {
       fields: ["name"]
     };
+    option (google.longrunning.operation_types) = {
```
Comment (andreamlin) [inline src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto]: Is it ok to not have a `response` for this void RPC method?

**Comment by andreamlin** [comment]
PTAL

File: src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto (hunk 3 above)
Comment (lukesneeringer) [reply src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto]: No.

Comment (lukesneeringer) [reply src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto]: (Use `google.protobuf.Empty`.)

Comment (andreamlin) [reply src/test/java/com/google/api/codegen/testsrc/libraryproto/annotated_no_config/library.proto]: Done.

**Pull request merged**